content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,264 @@
+
+![Federation](https://media.kuketz.de/blog/artikel/2020/welt-der-messenger/federation.png)
+
+## Events 
+
+As usual in this period of the year, the community of protocol makers united in the XMPP Summit. Two days of sharing and building the future of XMPP. This year has been productive, we covered various subjects such as:
+
+* XMPP Shortage Audit
+* Easy Passwordless Onboarding and Account Management
+* Account Rich Presence in PEP
+* MLS
+* End-to-End Encryption Key Management
+* Palaver IM Client
+* Why Push Notifications are not good enough?
+* MIX when?
+* IM-NG
+* commTeam ("Lightning talk")
+* Inbox / Unread / Bind2
+* XMPP as mandatory standard? (Lightning talk)
+* Stickers
+* XHTML-IM2 / Rich markup
+
+vanitasvitae has written a huge live thread on Mastodon:
+
+* [Day 1](https://fosstodon.org/@vanitasvitae/103571845414141817)
+* [Day 2](https://fosstodon.org/@vanitasvitae/103577121536634734)
+
+We will probably cover all these in more depth next month.
+
+## Software releases 
+          
+### Servers 
+
+The Ignite Realtime community has announced a series of updates:
+
+* [Hazelcast Openfire plugin 2.5.0](https://discourse.igniterealtime.org/t/hazelcast-openfire-plugin-2-5-0-released/86961)
+* [Openfire 4.5.0](https://discourse.igniterealtime.org/t/openfire-4-5-0-is-released/87004)
+* [XML Debugger Openfire plugin 1.7.3](https://discourse.igniterealtime.org/t/xml-debugger-openfire-plugin-1-7-3-released/87006)
+* [Monitoring Openfire plugin 2.0.0](https://discourse.igniterealtime.org/t/monitoring-openfire-plugin-2-0-0-released/87153)
+
+[Prosody 0.11.4](https://blog.prosody.im/prosody-0.11.4-released/) has been released.
+
+Erlang Solutions has released [MongooseIM 3.6.0](https://github.com/esl/MongooseIM/releases/tag/3.6.0) and [MongoosePush 2.0.0](https://github.com/esl/MongoosePush/releases/tag/2.0.0).
+
+### Clients and applications 
+
+The Dino team has announced [version 0.1](https://dino.im/blog/2020/01/dino-0.1-release/), with a "development process that started three years ago and already combined work of 30 contributors, including 4 summer of code students and multiple development sprints". The news has even been covered by [Hacker News](https://news.ycombinator.com/item?id=22185693) and [LinuxFr.org](https://linuxfr.org/news/sortie-de-dino-0-1)
+
+![Dino](https://dino.im/blog/2020/01/dino-0.1-release/main_window.png)
+
+Tigase has released [BeagleIM 3.5 and SiskinIM 5.5](https://tigase.net/beagleim-3.5-and-siskin-5.5-released/), respectively for macOS and iOS.
+
+![Beagle IM](https://tigase.net/static/28fe03d4f74066874cfcf0ca8f5b7732/fcda8/beagle-chat.png)
+
+JC Brand has announced the release of [Converse.js 6.0.0](https://github.com/conversejs/converse.js/releases/tag/v6.0.0).
+
+Georg Lukas has released  [yaxim 0.9.9 - FOSDEM Edition](http://yaxim.org/blog/2020/01/31/yaxim-0-dot-9-9-fosdem-edition/).
```

Comment:
I don't necessarily mind this, but I notice that for some projects, we refer to the "project team" that does a release, while for other projects, we refer to individuals (eg @jcbrand and @ge0rg). By referring to a person instead of a team, the project might be interpreted as a "hobby project" by people that read this. Maybe use a team name instead?

Reply:
I like this attitude. Also using "we" in the release notes post, so it would be consistent with that. You could also use passive speech, "[yaxim 0.9.9 - FOSDEM Edition](http://yaxim.org/blog/2020/01/31/yaxim-0-dot-9-9-fosdem-edition/) has been released."

Reply:
Oh, good point.
I did this way, for the sake of varying the grammar a little.
So I am not sure I understand the outcome of this discussion: should I use the team or the person form consistently?

Reply:
My preference is to _not_ use the person form. To vary the style, you could alternate between 'team' and passive speak, as ge0rg suggested.

Reply:
So we never mention the persons, even if one is a member of the team announcing the news?

Reply:
Varying the grammar is good, and at least for yaxim I'd be happy to see something like "the yaxim developers" instead of my name ;-)

Reply:
So, not clear to me.

Reply:
A continuity of this discussion would be interesting.